blockstore/splitstore/liveset.go

```diff
@@ -14,14 +14,14 @@ type LiveSet interface {
 	Close() error
 }
 
-var markBytes = []byte{}
+var markBytes []byte
```

Comment:
are you sure this is a good idea? I am concerned about the behaviour of the database backed ones if we pass nil.

Reply:
I'll revert just in case.

Reply:
thanks!